docs/elements/guides/sign-up.mdx

```diff
@@ -0,0 +1,235 @@
+---
+description: Learn how to build a complete sign-up form with Clerk Elements.
+---
+
+# Build a sign-up flow
```

Comment:
All my suggestions for this page are the same as my suggestions for Please use the suggestion change I made in [the sign-in guide](https://github.com/clerk/clerk-docs/pull/890/files/3bd5d665cc7641c0d2db48655b79b84bbe48cec0#diff-9addefb83500d3a49204d58b6636b2020b87608132d4aec8e1451e79cacefa25R2) above.

Once we've concluded discussions for those changes, I can pull this PR and add them to this file, saving us all time 👍

Reply:
Applied your suggestions to both guides 👍